index.js

```diff
@@ -44,8 +44,7 @@ queue.process('email', 30, require('./workers/email'));
 queue.process('sendBatchEmails', require('./workers/sendBatchEmails')(queue, db, baseUrl));
 queue.process('sendWeeklyRecapEmails', require('./workers/sendWeeklyRecapEmails')(queue, db, baseUrl));
 queue.process('amazonPayments', require('./workers/amazonPayments')(queue, db));
-
-queue.promote();
```

Comment:
Nah, let's leave it this way. if you can put back the queue.promote() line and we'll remove it once @TheHollidayInn 's branch is merged